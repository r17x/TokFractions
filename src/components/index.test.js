import React from 'react'
import {Input, ListDenominations} from '.'

import '@testing-library/react/cleanup-after-each'
import '@testing-library/jest-dom/extend-expect'
import {render} from '@testing-library/react'

test('Input Component', () => {
    const {container, getByLabelText} = render(<Input/>)
    const input = getByLabelText('input-text') 
    expect(input.value).toBe("")
    expect(input.type).toBe("text")
    expect(container).toMatchSnapshot()
})

test('ListDenominations Component', () => {
    const {container} = render(<ListDenominations data={[]}/>)
    expect(container).toMatchSnapshot()
})
