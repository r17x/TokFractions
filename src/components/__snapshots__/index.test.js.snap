// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component 1`] = `
<div>
  <input
    aria-label="input-text"
    class=""
    placeholder="Rp 500"
    type="text"
    value=""
  />
</div>
`;

exports[`ListDenominations Component 1`] = `
<div>
  <ul />
</div>
`;
